src/main.cpp

```diff
@@ -371,7 +371,7 @@ static std::unique_ptr<cMemorySettingsRepository> ParseArguments(int argc, char
 		// Parse the comand line args:
 		TCLAP::CmdLine cmd("Cuberite");
 		TCLAP::ValueArg<int> slotsArg    ("s", "max-players",         "Maximum number of slots for the server to use, overrides setting in setting.ini", false, -1, "number", cmd);
-		TCLAP::MultiArg<int> portsArg    ("p", "port",                "The port number the server should listen to", false, "port", cmd);
+		TCLAP::MultiArg<int> portsArg    ("p", "ports",                "The port numbers the server should listen to", false, "port", cmd);
```

Comment:
This should probably stay as `--port` as its a `MultiArg` - ie `./Cuberite --port 25565 --port 25566`, would need something added to the description to indicate it can be set multiple times.